cmd/entry.go

```diff
@@ -21,7 +22,7 @@ func (e *EntryCommand) Execute(args []string) error {
 		SubscriptionPrefix:        tuna.DefaultSubscriptionPrefix,
 		ReverseSubscriptionPrefix: tuna.DefaultSubscriptionPrefix,
 	}
-	err := tuna.ReadJSON(e.ConfigFile, config)
+	err := geo.ReadJSON(e.ConfigFile, config)
```

Comment:
It is not ideal to put `ReadJSON` into geo package because it's a common utility function. You can create a `util` package to put these common utility functions.